src/openforms/validations/api/views.py

```diff
@@ -58,14 +62,21 @@ class ValidationView(APIView):
     @extend_schema(
         operation_id="validation_run",
         summary=_("Validate value using validation plugin"),
-        request=ValidationInputSerializer,
+        request=extend_inline_serializer(
+            ValidationInputSerializer,
+            {
+                "value": serializers.CharField(
+                    label=_("value"), help_text=_("Value to be validated.")
+                )
```

Comment:
This was mistakenly set as `CharField`, but shouldn't `JSONField` do the trick? (as discussed a while ago on Slack)

Reply:
I don't think it will, I think it'll be turned into

```yaml
type: object
additionalProperties: true
```
because of drf-spectacular's handling:

```py
        if isinstance(field, serializers.JSONField):
            return append_meta(build_basic_type(OpenApiTypes.OBJECT), meta)
```
in `drf_spectacular.openapi.AutoSchema._map_serializer_field`

Reply:
Ah true, but isn't spectacular wrong in this case?

Reply:
yes :-) feel free to make an issue

It may also depend on the DB driver. Perhaps other databases only support objects for their JSON type? Might be good to raise an issue with drf-spectacular!